apps/web/lib/features/task/daily-plan/outstanding-all.tsx

```diff
@@ -0,0 +1,50 @@
+import { EmptyPlans } from 'lib/features/user-profile-plans';
+import { TaskCard } from '../task-card';
+import { useDailyPlan } from '@app/hooks';
+import { TaskEstimatedcount } from '.';
+
+interface OutstandingAll {
+    profile: any,
+}
+export function OutstandingAll({ profile }: OutstandingAll) {
+    const { outstandingPlans } = useDailyPlan();
+    const displayedTaskId = new Set();
+    return (
+        <div className="flex flex-col gap-6">
+            <TaskEstimatedcount outstandingPlans={outstandingPlans} />
+            {outstandingPlans?.length > 0 ? (
+                <>
+                    {outstandingPlans?.map((plan) => (
+                        <>
+                            {/* <PlanHeader plan={plan} planMode="Outstanding" /> */}
```

Comment:
delete if we don't use this code here anymore

Reply:
I merged with Squash, so we can delete it later in another PR @Cedric921 / @Innocent-Akim